manifests/server/conf.pp

```diff
@@ -4,12 +4,14 @@
 #
 # Regarding: POODLE - CVE-2014-3566
 #
-# The ``tls_cipher_suite`` parameter is set to ``HIGH:-SSLv2`` because OpenLDAP
-# cannot set the SSL provider natively.
+# Using module defaults and openldap-servers >= 2.4.40, a minimum bound of TLS
+# v1.2 will be set.  TLSv1, SSLv3, and SSLv2 ciphers will be removed from the
+# cipher suite.
 #
-# By default, it will run TLSv1 but cannot handle TLSv1.2 therefore the SSLv3
-# ciphers cannot be eliminated. Take care to ensure that your clients only
-# connect with TLSv1 if possible.
+# If openldap-servers is < 2.4.40, the ``tls_cipher_suite`` parameter will
+# default to ``HIGH:-SSLv2`` because OpenLDAP < 2.4.40 cannot ensure the SSL
+# provider natively. Take care to ensure that your clients only connect with
+# TLSv1 if possible.
```

Comment:
The code defaults to simplib::lookup('simp_options::openssl::cipher_suite' ,{ 'default_value' =>  ['DEFAULT', '!MEDIUM'] }) not HIGH:-SSLv2.  Which is correct (comment or code)?